engine/install/binaries.md

```diff
@@ -19,13 +21,14 @@ system's package management system to manage Docker installation and upgrades.
 Be aware that 32-bit static binary archives do not include the Docker daemon.
 
 Static binaries for the Docker daemon binary are only available for Linux (as
-`dockerd`). 
+`dockerd`) and Windows Server 2016 or Windows 10 (as `dockerd.exe`). 
```

Comment:
@thaJeztah Should we include WinServer 2019 to this list?

Reply:
Perhaps just "Windows" or "Window Server" 🤔. We should probably mention that;

- On Windows, these binaries only provide the ability to run native Windows containers (not Linux)
- On Windows 10/11 ("Windows client"), we recommend installing Docker Desktop

Reply:
For macOS, we may want to add a similar warning as well (also the link currently points to x86 packages, but we have downloads for m1 mac CLI's as well). It already _does_ describe it's a CLI only, but with recent confusing about "what's the CLI", we may want to add extra emphasis on that; explain that

- it doesn't come with a runtime, so it requires you to set up a functional engine elsewhere (in a VM, or a remote Linux machine)
- (currently) does not contain `buildx`, `docker scan`, nor `docker compose` 

For most users, Docker Desktop would be a more viable alternative.

Reply:
Added a note on this